### PR TITLE
Initial implementation for some scePower functions.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

**Comment by VelocityRa** [comment]
@petmac Maybe this process should be documented in the Wiki?

**Comment by petmac** [comment]
Agreed, it should. I mentioned "How module export stubs are generated from db.yml" in #51 which I should expand to cover this too.

File: src/emulator/modules/ScePower/src/ScePower.cpp [hunk 1]
```diff
@@ -140,31 +140,49 @@ EXPORT(int, scePowerRequestSuspend) {
...
+    if (freq < 0){
+        return SCE_POWER_ERROR_INVALID_VALUE;
+    }
+    return 0;
 }
 
-EXPORT(int, scePowerSetConfigurationMode) {
-    return unimplemented("scePowerSetConfigurationMode");
+EXPORT(int, scePowerSetConfigurationMode, int mode) {
+    if (mode != 0x80 && mode != 0x800 && mode != 0x10880){
```
Comment (petmac) [inline src/emulator/modules/ScePower/src/ScePower.cpp]: Do we have named constants for these values?

Comment (Rinnegatamante) [reply src/emulator/modules/ScePower/src/ScePower.cpp]: vita-headers doesn't have them. I grabbed those values with IDA Pro.

**Pull request merged**